### DISCUSSION TITLE
Help with PneumaticCraft Recipe JSON

### DISCUSSION THREAD

**Discussion by Garonne** [TeamPneumatic/pnc-repressurized#1165] [Q&A]
Is there a place where I can get help with my PneumaticCraft-related recipe JSON?
I try to convert one fluid into another one using a thermopneumatic processing plant, but no matter what I do, that recipe won't register.

Maybe someone can help me?

I'm sure the location is correct, because multiple recipe JSON files (also TPP) do work fine. Just this one won't work.

![image](https://user-images.githubusercontent.com/164475/236697311-49e3d9ae-65dc-4d4d-a7b1-37368119fcc3.png)

For clarity:
I've got Extreme Reactors installed, but for some reason (maybe Tinker's Construct?) Yellorium converts to molten uranium which is not accepted by the reactors. I want to convert it to yellorium fluid, but something seems to be wrong.
Can someone help me?

Or is there a different place to ask questions like these? (Discord perhaps? I didn't see a PneumaticCraft-related server...)

**Comment by desht** [comment]
There is a discord at https://discord.gg/Zf5XwbfBRj which is probably the best place for general questions like this.

Your recipe looks OK offhand. Have you verified that both fluids are using the correct ID?  Also, `forge:molten_uranium` looks like a fluid tag rather than a fluid. If that's the case, it should be `"tag": "forge:molten_uranium"`.
